### DISCUSSION TITLE
RPM repos

### DISCUSSION THREAD

**Discussion by workingoctopus** [DependencyTrack/dependency-track#2973] [Q&A] [answered]
Is it possible to check RPM components for updates (like npm/composer)? Is it possible to add RPM repo for such a purpose?

**Comment by syalioune** [comment, marked as answer]
Hello @workingoctopus 
That should be possible provided that you RPM repo match some requirements and a little development efforts. See discussion https://github.com/DependencyTrack/dependency-track/discussions/2801
